sequencing/ATACSeq/Rscripts/countsInPeaks.r

```diff
@@ -29,12 +29,14 @@ args <- commandArgs()
 configFile<-args[6]
 source(configFile)
 
-library(Rsubread)
-library(GenomicRanges)
-library(ChIPpeakAnno)
-library(ChIPseeker)
-library(TxDb.Hsapiens.UCSC.hg38.knownGene)
-library(org.Hs.eg.db)
+suppressWarnings(suppressPackageStartupMessages({
+  library(Rsubread)
+  library(GenomicRanges)
+  library(ChIPpeakAnno)
+  library(ChIPseeker)
+  library(TxDb.Hsapiens.UCSC.hg38.knownGene)
+  library(org.Hs.eg.db)
+}))
```

Comment:
@gtw99, A good alternative approach for suppressing unwanted package start up messages.